src/Symfony/Component/Validator/Constraints/Length.php

```diff
@@ -39,6 +39,9 @@ class Length extends Constraint
     public $max;
     public $min;
     public $charset = 'UTF-8';
+    public $ltrim;
```

Comment:
let's have some docblocks please

Reply:
Right now there's no Constraint classes with docblocks, but I will work on another PR to add it in all of those.

Reply:
We don't add docblocks on properties when they are strings (and here, names are pretty explicit anyway).

Reply:
This is boolean though. Name isn't explicit enough here. This could be also integer (number of chars to trim) or string (which chars to trim). But if that's your policy to not add docblocks here, ok then..